ush/config_defaults.yaml

```diff
@@ -648,7 +648,7 @@ workflow:
   FV3_NML_BASE_SUITE_FN: "input.nml.FV3"
   FV3_NML_YAML_CONFIG_FN: "FV3.input.yml"
   FV3_NML_BASE_ENS_FN: "input.nml.base_ens"
-  FV3_NML_FN: "input.nml"
+  FV3_NML_FN: '{{ FV3_NML_BASE_SUITE_FN[:-4] }}'
```

Comment:
This is a pretty dangerous assumption that `FV3_NML_BASE_SUITE_FN` won't change its suffix. Why not base all the others off the `FV3_NML_FN` since it's likely the most stable and is the requirement from the model? And even if it does change, since it's set by the model, we'll likely want to rename all of them. 

It could also be fine to leave it like it is.

This is what I mean when I suggest using `FV3_NML_FN` as the source of information:
```
FV3_NML_BASE_SUITE_FN: '{{ FV3_NML_FN }}.FV3'
FV3_NML_FN: "input.nml"
```